wapitiCore/attack/mod_http_headers.py

```diff
@@ -45,19 +45,19 @@ class ModuleHttpHeaders(Attack):
             "list": check_list_xframe,
             "info": {"error": XFRAME_OPTIONS_NOT_SET, "warning": INVALID_XFRAME_OPTIONS},
             "log": "Checking X-Frame-Options:",
-            "wstg": WSTG_CODE_FRAME_OPTIONS
+            "finding": ClickjackingFinding,
```

Comment:
do we need more than wstg code here ? why not keeping `"wstg": ClickjackingFinding.code()`

Reply:
Everything is held by the finding class, no need to add another parameter